odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -0,0 +1,1217 @@
+            # read the definition on the parent
+            'SELECT "test_new_api_discussion"."id" AS "id", "test_new_api_discussion"."message_properties" AS "message_properties" FROM "test_new_api_discussion" WHERE "test_new_api_discussion".id IN %s',
+            # check the many2one existence
+            'SELECT "res_partner".id FROM "res_partner" WHERE "res_partner".id IN %s',
+            'SELECT "res_partner"."id" AS "id", "res_partner"."message_main_attachment_id" AS "message_main_attachment_id", "res_partner"."email_normalized" AS "email_normalized", "res_partner"."message_bounce" AS "message_bounce", "res_partner"."name" AS "name", "res_partner"."display_name" AS "display_name", "res_partner"."date" AS "date", "res_partner"."title" AS "title", "res_partner"."parent_id" AS "parent_id", "res_partner"."ref" AS "ref", "res_partner"."lang" AS "lang", "res_partner"."tz" AS "tz", "res_partner"."user_id" AS "user_id", "res_partner"."vat" AS "vat", "res_partner"."company_registry" AS "company_registry", "res_partner"."website" AS "website", "res_partner"."comment" AS "comment", "res_partner"."active" AS "active", "res_partner"."employee" AS "employee", "res_partner"."function" AS "function", "res_partner"."type" AS "type", "res_partner"."street" AS "street", "res_partner"."street2" AS "street2", "res_partner"."zip" AS "zip", "res_partner"."city" AS "city", "res_partner"."state_id" AS "state_id", "res_partner"."country_id" AS "country_id", "res_partner"."partner_latitude" AS "partner_latitude", "res_partner"."partner_longitude" AS "partner_longitude", "res_partner"."email" AS "email", "res_partner"."phone" AS "phone", "res_partner"."mobile" AS "mobile", "res_partner"."is_company" AS "is_company", "res_partner"."industry_id" AS "industry_id", "res_partner"."company_id" AS "company_id", "res_partner"."color" AS "color", "res_partner"."partner_share" AS "partner_share", "res_partner"."commercial_partner_id" AS "commercial_partner_id", "res_partner"."commercial_company_name" AS "commercial_company_name", "res_partner"."company_name" AS "company_name", "res_partner"."create_uid" AS "create_uid", "res_partner"."create_date" AS "create_date", "res_partner"."write_uid" AS "write_uid", "res_partner"."write_date" AS "write_date", "res_partner"."contact_address_complete" AS "contact_address_complete", "res_partner"."signup_token" AS "signup_token", "res_partner"."signup_type" AS "signup_type", "res_partner"."signup_expiration" AS "signup_expiration", "res_partner"."ocn_token" AS "ocn_token", "res_partner"."partner_gid" AS "partner_gid", "res_partner"."additional_info" AS "additional_info", "res_partner"."phone_sanitized" AS "phone_sanitized" FROM "res_partner" WHERE "res_partner".id IN %s',
```

Comment:
This is clearly dependent on which other modules are installed at the time of the test.
In other words: this is not testable outside the runbot, and will need to be fixed all the time.
You should make add a specific model in the module, and not rely on `res.partner`.